src/administration/web/configure-environment.md

```diff
@@ -12,10 +12,10 @@ environment.
 
 ![Configure Platform.sh environment settings](/images/ui-conf-environment-settings.png)
 
-The *Delete* action will destroy all services running on this environment (PHP, MySQL, volumes...)
+The *Delete* action will destroy all services running on this environment (PHP, MySQL, volumes, ...)
```

Comment:
I'm not sure if the comma is correct there.  However, it's probably better to instead use "volumes, etc." rather than the ellipsis anyway.